tools/distpartitioning/README.md

```diff
@@ -0,0 +1,40 @@
+### xxx_nodes.txt format
+This file is used to provide node information to this framework. Following is the format for each line in this file:
+```
+<node_type> <weight1> <weight2> <weight3> <weight4> <global_type_node_id> <attributes>
+```
+where node_type is the type id of this node, weights can be any number of columns as determined by the user, type_node_id is the `locally` contiguous ids starting from `0` for a particular node_type. And attributes can be any number of columns at the end of each line. 
```

Comment:
where is `type_node_id`?

Reply:
why is the format here different from the format in 
https://doc.dgl.ai/guide/distributed-preprocessing.html#run-parmetis-and-output-formats

Reply:
in your text, you mean `type_node_id` and in the format you mention `global_type_node_id`. they seem different. what is `global_type_node_id`?

Reply:
Same question where is `type_node_id`?